app/types.ts

```diff
@@ -113,6 +114,7 @@ export type RatePayload = {
   token: string;
   amount?: number;
   currency: string;
+  provider_id?: string;
```

Comment:
this should be `providerId` not `provider_id`

Reply:
Updated provider_id to providerId